packages/shorebird_cli/integration_test/shorebird_cli_integration_test.dart

```diff
@@ -32,11 +32,15 @@ R runWithOverrides<R>(R Function() body) {
 }
 
 void main() {
+  final shorebirdHostedURL = Platform.environment['SHOREBIRD_HOSTED_URL'];
```

Comment:
Not really related to fixing the tests, but I got confused when I tried to run the tests and it failed because of null check. Made this small change to make it clear why it is failing when the variable is not set.